tmail-backend/apps/distributed/src/main/java/com/linagora/tmail/james/app/DistributedEmailAddressContactEventDeadLettersModule.java

```diff
@@ -0,0 +1,28 @@
+package com.linagora.tmail.james.app;
+
+
+import org.apache.james.events.CassandraEventDeadLetters;
+import org.apache.james.events.CassandraEventDeadLettersDAO;
+import org.apache.james.events.CassandraEventDeadLettersGroupDAO;
+import org.apache.james.events.EventDeadLetters;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import com.google.inject.name.Named;
+import com.linagora.tmail.james.jmap.EmailAddressContactInjectKeys;
+import com.linagora.tmail.james.jmap.contact.TmailJmapEventSerializer;
+
+public class DistributedEmailAddressContactEventDeadLettersModule extends AbstractModule {
+
+    @Provides
+    @Singleton
+    @Named(EmailAddressContactInjectKeys.AUTOCOMPLETE)
+    EventDeadLetters provideEventDeadLetters(CassandraEventDeadLettersGroupDAO cassandraEventDeadLettersGroupDAO,
```

Comment:
Why not adding this block in `DistributedEmailAddressContactEventModule` directly? Do we really need to create an extra module just for that?

Reply:
I rebased master yesterday :(
Can we do it next time?

Reply:
DistributedEmailAddressContactEventModule is used in both cassandra and postgres
We  should not fixed Cassandra for `DistributedEmailAddressContactEventModule`

Reply:
yes,
my meaning is should not import `Cassandra***` in this class